packages/ws-worker/src/events.ts

```diff
@@ -8,7 +8,7 @@ export type ClaimPayload = { demand?: number };
 export type ClaimReply = { attempts: Array<ClaimAttempt> };
 export type ClaimAttempt = { id: string; token: string };
 
-export const GET_ATTEMPT = 'fetch:attempt';
+export const GET_RUN = 'fetch:run';
```

Comment:
TODO: I think `get run` is the wrong semantics here - this ought to change to `GET_PLAN`. There will be a bunch of matching renames.